lib/Driver/ToolChains/Hcc.cpp

```diff
@@ -322,6 +324,61 @@ void HCC::CXXAMPLink::ConstructJob(
         CmdArgs.push_back("-lstdc++");
     }
 
+    if (getenv("HCC_BUILD")) {
+        CmdArgs.push_back("-L" CMAKE_BUILD_LIB_DIR);
+        CmdArgs.push_back("--rpath=" CMAKE_BUILD_LIB_DIR);
+
+        CmdArgs.push_back("-L" CMAKE_BUILD_COMPILER_RT_LIB_DIR);
+        CmdArgs.push_back("--rpath=" CMAKE_BUILD_COMPILER_RT_LIB_DIR);
+    }
+    else {
+        std::string path_hcc_lib;
+
+        if (const char *p = getenv("HCC_HOME")) {
+            path_hcc_lib = std::string(p) + std::string("/lib");
+        }
+        else {
+            path_hcc_lib = C.getDriver().Dir + "/../../" + CMAKE_INSTALL_LIB;
+        }
+
+        HCCLibPath = "-L" + path_hcc_lib;
+        CmdArgs.push_back(HCCLibPath.c_str());
+
+        HCCRLibPath = "--rpath=" + path_hcc_lib;
+        CmdArgs.push_back(HCCRLibPath.c_str());
+    }
+
+    if (getenv("HCC_GTEST")) {
```

Comment:
Where do we need this?  Can we remove it from the compiler?

Reply:
In hcc-config we have a --gtest flag that would add these options. If we don't need to support it, then I can remove it from the compiler.